docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -53,12 +53,20 @@ Change the first line of `tomcat/bin/catalina.bat`.
 ```shell
 set "CATALINA_OPTS=-javaagent:/path/to/skywalking-agent/skywalking-agent.jar"
 ```
+
 - JAR file  
 Add `-javaagent` argument to command line in which you start your app. eg:
  ```shell
  java -javaagent:/path/to/skywalking-agent/skywalking-agent.jar -jar yourApp.jar
  ```
- 
+
+- Jetty 
+Modify `jetty.sh`.
```

Comment:
From the review page, https://github.com/apache/skywalking/blob/ad70d42674fc5d7c1448a415948a49043010d60a/docs/en/setup/service-agent/java-agent/README.md 
The format is not right. Please fix.